gateway-server/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+  application:
+    name: gateway-server
+  cloud:
+    gateway:
+      discovery:
+        locator:
+          enabled: true
+          lower-case-service-id: true
+
+server:
+  port: 8072
+
+eureka:
+  client:
+    registerWithEureka: false
+    fetch-registry: true
```

Comment:
Isn't this the default with the eureka client anyway?

Reply:
yes